.env

```diff
@@ -1,4 +1,4 @@
 MYSQL_HOST=localhost
 MYSQL_USER=root
-MYSQL_PASS=
+MYSQL_PASS=12345678
```

Comment:
make sure you have .env included in your git-ignore file (so that these personal changes dont get included)